application/apps/indexer/session/src/handlers/search_values.rs

```diff
@@ -28,6 +31,22 @@ pub async fn execute_value_search(
     state: SessionStateAPI,
 ) -> OperationResult<()> {
     debug!("RUST: Search values operation is requested");
+    let invalid = filters
+        .iter()
+        .filter(|f| !is_valid(f))
+        .cloned()
+        .collect::<Vec<String>>();
+    if !invalid.is_empty() {
+        Err(NativeError {
+            severity: Severity::ERROR,
+            kind: NativeErrorKind::OperationSearch,
+            message: Some(format!(
+                "Next {} filter(s) are invalid: {}",
```

Comment:
same thing. if you put it to as a function to the searchers module, you can call the check on both the search-holder and value-holder setup

Reply:
that way you also do not have to make the value field public API since you just need it inside the process crate

Reply:
Actually I've missed issue with handling error on setup... it should put back holder, but doesn't